geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -183,20 +184,15 @@ class SelectionMapFragment(
         }
 
         map.setGpsLocationEnabled(true)
-        map.setGpsLocationListener { point -> onLocationChanged(point) }
-
-        previousState?.let { restoreZoomFromPreviousState(it) }
 
         map.setFeatureClickListener(::onFeatureClicked)
         map.setClickListener { onClick() }
 
         selectionMapData.getMappableItems().observe(viewLifecycleOwner) {
             updateItems(it)
             updateCounts(binding)
-        }
 
-        selectedFeatureViewModel.getSelectedFeatureId()?.let {
```

Comment:
I think this is redundant. `updateItems` used to do this work. Now `initializeViewport` does.